added status window and updated spacedust

Yvonne-Aizawa opened the pull request:
It is not done quite yet but i will push it at least. 
Here are a few things that still need fixing 
- [X] fix leaderboard stats so they are not cut off 
- [x]  links within announcements should be clickable
- [X] fix cut off by using a scroll bar

this would also close #29 since it uses features from it

Yvonne-Aizawa committed:
added status window and updated spacedust

Allstreamer commented:
Consider Adding a scroll view to fix the cut off, unless it's horizontal cut-off then you may wanna restructure it

Yvonne-Aizawa committed:
small fixes

fixed that leaderboard info would be cut off horizontally
added a scroll bar to limit the height of the text to prevent it from taking up the entire secreen

Yvonne-Aizawa committed:
updated spacedust and improved anouncement list

spacedust updated to 1.0.9
annoucement list is collapsible now

Yvonne-Aizawa committed:
url's are now displayed in announcements and are clickable

when there is an url in a announcement you can click it

Yvonne-Aizawa committed:
Delete .vscode directory

Yvonne-Aizawa committed:
updated gitignore to prevent vscode dir from being uploaded

Allstreamer reviewed (approved):
Looks good (you can force push and skip ci, if you want to, since the .git ignore change shouldn't break anything that wasn't already verified)
